packages/generator-langium/templates/packages/cli/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+        "rootDir": "src",
+        "outDir": "out",
+        "declarationDir": "out"
```

Comment:
Is this option required? Since it's identical to `outDir`, I think it could be removed.

This also goes here:

- https://github.com/eclipse-langium/langium/blob/89319417f3a6a1cea24127a57e93e4f76e4189fc/packages/generator-langium/templates/packages/language/tsconfig.src.json
- https://github.com/eclipse-langium/langium/blob/89319417f3a6a1cea24127a57e93e4f76e4189fc/packages/generator-langium/templates/packages/web/tsconfig.json

Reply:
Yes, that is superfluous if not other directory is required. Will change.

Reply:
Done ⬇️

Reply:
Still has the `declarationDir` option. Same goes here: https://github.com/eclipse-langium/langium/blob/kaisalmen/issue-1495/packages/generator-langium/templates/packages/web/tsconfig.json